enigmatry-entry-blueprint-app/package.json

```diff
@@ -20,43 +20,43 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^17.3.9",
-    "@angular/cdk": "^17.3.9",
-    "@angular/common": "^17.3.9",
-    "@angular/compiler": "^17.3.9",
-    "@angular/core": "^17.3.9",
-    "@angular/forms": "^17.3.9",
-    "@angular/material": "^17.3.9",
-    "@angular/material-date-fns-adapter": "^17.3.9",
-    "@angular/platform-browser": "^17.3.9",
-    "@angular/platform-browser-dynamic": "^17.3.9",
-    "@angular/router": "^17.3.9",
+    "@angular/animations": "^18.2.13",
+    "@angular/cdk": "^18.2.14",
+    "@angular/common": "^18.2.13",
+    "@angular/compiler": "^18.2.13",
+    "@angular/core": "^18.2.13",
+    "@angular/forms": "^18.2.13",
+    "@angular/material": "^18.2.14",
+    "@angular/material-date-fns-adapter": "^18.2.14",
+    "@angular/platform-browser": "^18.2.13",
+    "@angular/platform-browser-dynamic": "^18.2.13",
+    "@angular/router": "^18.2.13",
     "@azure/msal-browser": "3.2.0",
-    "@enigmatry/entry-components": "17.0.0",
-    "@enigmatry/entry-form": "17.0.0",
-    "@microsoft/applicationinsights-web": "3.1.0",
-    "@ngx-formly/core": "6.3.1",
-    "@ngx-formly/material": "6.3.1",
-    "date-fns": "^2.30.0",
+    "@enigmatry/entry-components": "18.0.0-preview.5",
```

Comment:
I would merge this with the preview to master. So that other teams can start the transition.

Reply:
We could release 18.0.0 of angular building blocks, after that PR is reviewed.